### DISCUSSION TITLE
Notification action triggering all the time

### DISCUSSION THREAD

**Discussion by LemonPie22** [filamentphp/filament#12233] [Help]
### Package

Panel builder

### Package Version

v3.2.61

### How can we help you?

I want to send a welcome email to new users as they are added, but only after confirmation has been given.

My code in `CreateUser` class:
```
protected function afterCreate(): void
{
    Notification::make()
        ->title('Send welcome email?')
        ->success()
        ->body('Do you want to send a welcome email message to this user?')
        ->persistent()
        ->actions([
            Action::make('yes')
                ->button()
                ->action($this->record->notify(new Welcome($this->getRecord())))
                ->close(),
            Action::make('no')
                ->color('gray')
                ->close(),
        ])
        ->send();
}
```

When I add a new user modal pops up as expected:

<img width="388" alt="Screenshot 2024-04-07 at 13 43 41" src="https://github.com/filamentphp/filament/assets/107488019/62a1cc59-002b-447d-82d1-fb74946d401f">

But the email goes out whether I click "Yes" or "No" on the modal. In fact it goes out before I respond to the modal at all.

How can I control sending the email?

**Comment by torgodly** [comment]
You should create a [custom notification](https://filamentphp.com/docs/3.x/notifications/customizing-notifications#using-a-custom-notification-view) for this. The Notifications Actions are currently designed for marking as read or opening a URL on click, but the actions will be triggered automatically.

**Reply by LemonPie22** [reply]
That seems quite complicated. Is there a simpler way to get confirmation before sending the email?
